plenum/test/pool_transactions/helper.py

```diff
@@ -96,18 +115,61 @@ def addNewNode(looper, stewardClient, stewardWallet, newNodeName, tdir, tconf,
     waitForSufficientRepliesForRequests(looper, stewardClient,
                                         requests=[req])
 
-    # initNodeKeysForBothStacks(newNodeName, tdir, sigseed, override=True)
-    # node = nodeClass(newNodeName, basedirpath=tdir, config=tconf,
-    #                  ha=(nodeIp, nodePort), cliha=(clientIp, clientPort),
-    #                  pluginPaths=allPluginsPath)
-    # if autoStart:
-    #     looper.add(node)
-    # return node
     return start_newly_added_node(looper, newNodeName, tdir, sigseed,
                                   (nodeIp, nodePort), (clientIp, clientPort),
                                   tconf, autoStart, allPluginsPath, nodeClass)
 
 
+def start_not_added_node(looper,
+                         tdir, tconf, allPluginsPath,
+                         newNodeName):
+    '''
+    Creates and starts a new node, but doesn't add it to the Pool
+    (so, NODE txn is not sent).
+    '''
+    sigseed, verkey, bls_key, nodeIp, nodePort, clientIp, clientPort = \
+        prepare_new_node_data(tconf, tdir, newNodeName)
+
+    new_node = start_newly_added_node(looper, newNodeName,
+                                      tdir, randomString(32).encode(),
+                                      (nodeIp, nodePort), (clientIp, clientPort),
+                                      tconf, True, allPluginsPath, TestNode)
+    return sigseed, bls_key, new_node
+
+
+def add_started_node(looper,
+                     new_node,
+                     txnPoolNodeSet,
+                     client_tdir,
+                     stewardClient, stewardWallet,
+                     sigseed,
+                     bls_key):
+    '''
+    Adds already created node to the pool,
+    that is sends NODE txn.
+    Makes sure that node is actually added and connected to all otehr nodes.
+    '''
+    newSteward, newStewardWallet = addNewSteward(looper, client_tdir,
+                                                 stewardClient, stewardWallet,
+                                                 "Steward" + new_node.name,
+                                                 clientClass=TestClient)
+    node_name = new_node.name
+    send_new_node_txn(sigseed,
+                      new_node.poolManager.nodeReg[node_name][0],
+                      new_node.poolManager.nodeReg[node_name][1],
+                      new_node.poolManager.cliNodeReg[node_name + "C"][0],
+                      new_node.poolManager.cliNodeReg[node_name + "C"][1],
+                      bls_key,
+                      node_name,
+                      newSteward, newStewardWallet)
+
+    txnPoolNodeSet.append(new_node)
+    looper.run(checkNodesConnected(txnPoolNodeSet))
+    ensureClientConnectedToNodesAndPoolLedgerSame(looper, newSteward, *txnPoolNodeSet)
+
+    waitNodeDataEquality(looper, new_node, *txnPoolNodeSet[:-1])
+
+
 def start_newly_added_node(
```

Comment:
This function should be renamed since it is used now in both cases: for starting an added node and for starting a not yet added node.